app/models/pglogical_subscription.rb

```diff
@@ -86,16 +86,24 @@ def pglogical(refresh = false)
     self.class.pglogical(refresh)
   end
 
-  def self.validate(host, port, user, password, dbname, id)
+  def self.validate(host: nil, port: nil, user: nil, password: nil, dbname: nil, id: nil)
```

Comment:
@carbonin this signature does not look pretty, but i am not sure how it could be organized better...